src/Microsoft.AspNet.Razor/TagHelpers/TagHelperAttributeDescriptor.cs

```diff
@@ -46,13 +82,15 @@ internal TagHelperAttributeDescriptor([NotNull] string name, [NotNull] PropertyI
             [NotNull] string name,
             [NotNull] string propertyName,
             [NotNull] string typeName,
-            bool isIndexer)
+            bool isIndexer,
+            TagHelperUseageDescriptor useage)
```

Comment:
Parameter and property need longer names. suggest `usageDescriptor`. Same in `TagHelperDescriptor`.